packages/app-mobile/components/SelectDateTimeDialog.tsx

```diff
@@ -157,11 +157,11 @@ export default class SelectDateTimeDialog extends React.PureComponent<any, any>
 				}}
 			>
 				<View style={{ ...styles.modalView, backgroundColor: theme.backgroundColor }}>
-					<View style={{ padding: 15, flexBasis: 'auto', paddingBottom: 0, flexGrow: 0, width: '100%', borderBottomWidth: 1, borderBottomColor: theme.dividerColor, borderBottomStyle: 'solid' }}>
+					<View style={{ padding: 15, flexBasis: 'auto', paddingBottom: 0, flexGrow: 0, width: '100%', borderBottomWidth: 1, borderBottomColor: theme.dividerColor }}>
```

Comment:
This resolves an error detected by TypeScript after converting a `require` to an `import` &mdash; `borderBottomStyle` is not a valid style property. (However, [borderStyle](https://reactnative.dev/docs/0.74/view-style-props#borderstyle) is.)

See also [the React Native `ViewStyle` documentation](https://reactnative.dev/docs/view-style-props).